components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3737,13 +3750,26 @@ where
                             .set_flags(WriteBatchFlags::TRANSFER_LEADER_PROPOSAL.bits());
                         cmd.mut_admin_request()
                             .set_cmd_type(AdminCmdType::TransferLeader);
-                        cmd.mut_admin_request().mut_transfer_leader().set_peer(from);
+                        cmd.mut_admin_request()
+                            .mut_transfer_leader()
+                            .set_peer(from.clone());
                         self.propose_raft_command(
                             cmd,
                             Callback::None,
                             DiskFullOpt::AllowedOnAlmostFull,
                         );
+                        info!("after propose transfer leader command";
```

Comment:
Redundant here.

The above log `propose transfer leader command` has already shown that the leader needs to send the `TransferLeader` cmd.

Reply:
This PR encountered some other issues during testing, and I temporarily added some logs. I will remove them later, so please pause the review for now.